python/pyspark/ml/regression.py

```diff
@@ -454,7 +466,7 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeModel(JavaModel):
+class DecisionTreeModel(JavaModel, MLWritable, MLReadable):
```

Comment:
Oops, I didn't spot this.  This class should be left alone, and DecisionTreeRegressionModel can be modified instead.